README.md

```diff
@@ -19,9 +19,50 @@ with `3`, e.g. `pip3` and `python3`, or follow the process for making Python
 
 ## Cloning and running it locally
 
+**Linux deployment:**
+
+Optional installation guides:
```

Comment:
I'm not sure why this says "optional installation guides"... they basically need to install it all if it's not already installed. I'd say this line can go away.

For the lines below, can they be formatted as numbered lists like the steps above and below this? I think that'd make it a bit clearer.